src/v2/Components/ArtworkFilter/ArtworkFilters/PriceRangeFilterNew.tsx

```diff
@@ -0,0 +1,211 @@
+import { FC, useEffect, useState, useMemo, FormEvent } from "react"
+import { Box, Flex, LabeledInput, Spacer, Text } from "@artsy/palette"
+import {
+  SelectedFiltersCountsLabels,
+  useArtworkFilterContext,
+} from "../ArtworkFilterContext"
+import styled from "styled-components"
+import { themeGet } from "@styled-system/theme-get"
+import { FilterExpandable } from "./FilterExpandable"
+import { isCustomValue } from "./Utils/isCustomValue"
+import { useFilterLabelCountByKey } from "../Utils/useFilterLabelCountByKey"
+import { Range, RANGE_DOT_SIZE } from "v2/Components/Range"
+import { debounce } from "lodash"
+
+// Disables arrows in numeric inputs
+export const NumericInput = styled(LabeledInput).attrs({ type: "number" })`
```

Comment:
I'd suggest moving this NumericInput component below the main `PriceFilterNewComponent` since its mostly just CSS and is secondary to the file's main concern.